packages/react-core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -50,6 +50,8 @@ export interface DatePickerProps
   isDisabled?: boolean;
   /** Error message to display when the text input contains a non-empty value in an invalid format. */
   invalidFormatText?: string;
+  /** Error message to display when the text input is empty and the isRequired prop is also passed in. */
```

Comment:
you should nit need this.  you cna use the one in the object on line 20.

Reply:
sure, I forgot to delete that prop